src/estimators/tests/exact_set_test.py

```diff
@@ -22,43 +22,66 @@
 import numpy as np
 
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import AddRandomElementsNoiser
-from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactSet
+from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactMultiSet
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import LessOneEstimator
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import LosslessEstimator
 
 
-class ExactSetTest(absltest.TestCase):
+class ExactMultiSetTest(absltest.TestCase):
 
-  def test_sketch(self):
-    s = ExactSet()
+  def test_sketch_for_exact_multi_set(self):
+    s = ExactMultiSet()
     s.add_ids([1, 2])
     self.assertLen(s, 2)
     self.assertIn(2, s)
     self.assertNotIn(3, s)
```

Comment:
Hi Matthew, would you mind adding error messages to these three assertions so that it is easier for debugging.